mod-source-record-storage-server/src/main/resources/liquibase/tenant/scripts/v-0.0.2/2020-09-09--15-00-fill-instance-hrid.xml

```diff
@@ -0,0 +1,22 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+
+  <changeSet id="2020-09-09--15-00-fill-instance-hrid" author="VolodymyrRohach">
+    <preConditions onFail="MARK_RAN">
+      <tableExists tableName="records_lb" schemaName="${database.defaultSchemaName}"/>
+    </preConditions>
+    <sql>
+      UPDATE ${database.defaultSchemaName}.records_lb
+      SET instance_hrid =
+      (SELECT fields
+      FROM ${database.defaultSchemaName}.marc_records_lb
+      CROSS JOIN LATERAL jsonb_array_elements((marc_records_lb.content #>> '{}')::jsonb -> 'fields') AS fields
+      WHERE fields ?? '001' AND records_lb.id = ${database.defaultSchemaName}.marc_records_lb.id) ->> '001'
+      FROM  ${database.defaultSchemaName}.marc_records_lb;
```

Comment:
add please condition to update records where instance_id is present

Reply:
Done.